Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/InkToolbar/InkToolbarIsStencilButtonCheckedChangedEventArgs.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security;
+using Microsoft.Windows.Interop;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    /// <summary>
+    /// Provides data for events. This class cannot be inherited.
+    /// </summary>
+    /// <remarks>Copy from <see cref="global::Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
+    /// <seealso cref="global::Windows.UI.Xaml.Controls.InkToolbarIsStencilButtonCheckedChangedEventArgs"/>
+    public sealed class InkToolbarIsStencilButtonCheckedChangedEventArgs : EventArgs
+    {
+        [SecurityCritical]
```

Comment:
Why is this security critical?

Reply:
Another point I wanted reviewed.  It's a replicated pattern I grabbed from other examples.  I'll pull them out.